src/components/common/RouterLinkNetwork.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <router-link
+    :class="cclass"
+    :to="url"
+    :exact="exact"
+    :title="title"
+    @click.native="handleClick()"
+  >
+    <slot></slot>
+  </router-link>
+</template>
+
+<script>
+import { mapGetters } from "vuex"
+export default {
+  name: `routerLinkNetwork`,
+  props: {
+    cclass: {
+      type: String,
+      default: null
+    },
+    to: {
+      type: String,
+      default: null
+    },
+    exact: {
+      type: String,
+      default: null
+    },
+    title: {
+      type: String,
+      default: null
+    }
+  },
+  computed: {
+    ...mapGetters([`network`]),
+    url() {
+      if (this.network.length) {
+        return `/${this.network}${this.to}`
```

Comment:
can we remove `-mainnet` from the link? so just `/terra/` or `/cosmos/`?

for testnets, we can leave `-testnet` in.

Reply:
I just left `cosmos-hub` and yes, `terra` for terra-mainnet. But I used slice, so you might not like it xD (please don't use the vomit emoji)

Reply:
Rewritten a bit